### ISSUE TITLE
Extension install package should handle .Net Framework 4.6 (Preview)

### ISSUE THREAD

**Issue by zmira** [bridgedotnet/Bridge#187]
Hi.

While attempting to update from 1.1 to 1.2, the install package complained the required framework was not installed in the system. Since I have had this problem with other packages and I have .Net Framework 4.6 installed on my machine, I decided to take a look.

After extracting the install package to a folder and going through the *extension.vsixmanifest* I noticed that, to fix this, I only needed to change the dependencies section in the file to
```xml
  <Dependencies>
    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" Version="[4.5,)" />
    <Dependency Id="Microsoft.VisualStudio.MPF.11.0" DisplayName="Visual Studio MPF 11.0" Version="[11.0]" />
  </Dependencies>
```
After re-creating the package file, it installed without any further issues.

Please note the **Version** attributes (for reference, see [VSIX Extension Schema 2.0 Reference](https://msdn.microsoft.com/en-us/library/hh696828.aspx)) and their changed values.

Thanks.

**Comment by geoffreymcgill** [comment]
Hi @zmira, 

Thanks for pointing out the problem. We will fix for the v1.3 release, which is do to release on Monday.

**Comment by DaniilVeriga** [comment]
@zmira, v1.3 has been released. We applied your suggestion. Thank you again.
